subtitles/vosk_transcriber.py

```diff
@@ -1,6 +1,11 @@
 import subprocess
+
+import vosk
 from vosk import Model, KaldiRecognizer, SetLogLevel
 from transcriber import Transcriber
+from threading import Lock
+
+mutex_vosk = Lock()
```

Comment:
Since this is treated as a constant, I would suggest the following naming:

```suggestion
VOSK_MUTEX = Lock()
```

Reply:
As @nshmyrev pointed out, once you move the creation of the transcribers inside the thread no problems arise. Hence, no mutex is needed and everything works fine.